src/main/java/com/krech/botv3/controller/WordController.java

```diff
@@ -0,0 +1,78 @@
+package com.krech.botv3.controller;
+
+import com.krech.botv3.domain.WordObject;
+import com.krech.botv3.domain.rest.request.RestWordObject;
+import com.krech.botv3.repository.IndexRepository;
+import com.krech.botv3.repository.WordRepository;
+import com.krech.botv3.service.WordService;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.*;
+
+import java.util.List;
+
+@RestController
+public class WordController {
+
+
+    private final WordService wordService;
+    private WordRepository wordRepository;
+    private IndexRepository indexRepository;
+
+    @Autowired
+    public WordController(WordService wordService, WordRepository wordRepository, IndexRepository indexRepository) {
+        this.wordService = wordService;
+        this.wordRepository = wordRepository;
+        this.indexRepository = indexRepository;
+    }
+
+
+    @PostMapping(value = "/word/create")
+    public ResponseEntity<?> saveOneWord(@RequestBody RestWordObject restWordObject) {
```

Comment:
First check, if the word already exists. If it is return bad request with description of error.